inc/admin/dashboard/class-networkdashboard.php

```diff
@@ -55,4 +56,75 @@ protected function getTotalNumberOfBooks(): int {
 			)
 		);
 	}
+
+	protected function getNetworkChecklist() : array {
+		$items = [
+			[
+				'title' => __( 'Brand your network', 'pressbooks' ),
+				'link' => admin_url( 'customize.php' ),
+				'description' => __( 'Display your institutional logos and colors across your network', 'pressbooks' ),
+				'checked' => get_network_option( null, 'network_checklist_pressbooks_branding', false ),
+			],
+			[
+				'title' => __( 'Customize your homepage', 'pressbooks' ),
+				'link' => admin_url( 'edit.php?post_type=page' ),
+				'description' => __( 'Edit the textboxes and menu links on your homepage' ),
+				'checked' => get_network_option( null, 'network_checklist_customize_homepage', false ),
+			],
+			[
+				'title' => __( 'Review network settings', 'pressbooks' ),
+				'link' => network_admin_url( 'admin.php?page=pb_network_analytics_options' ),
+				'description' => __( 'Adjust defaults for new books and user permissions ' ),
+				'checked' => get_network_option( null, 'network_checklist_review_network', false ),
+			],
+			[
+				'title' => __( 'Configure Google Analytics', 'pressbooks' ),
+				'link' => network_admin_url( 'settings.php?page=pb_network_analytics_options#tabs-3' ),
+				'description' => __( 'Get more insight into your network’s web traffic' ),
+				'checked' => get_network_option( null, 'network_checklist_google_analytics', false ),
+			],
+		];
+
+		// Check if either SSO plugin is activated
+		if ( is_plugin_active( 'pressbooks-saml-sso/pressbooks-saml-sso.php' ) ) {
+			$sso_item = [
+				'title' => __( 'Configure Single Sign On (SSO)', 'pressbooks' ),
+				'link' => network_admin_url( 'admin.php?page=pb_saml_admin' ),
+				'description' => __( 'Allow users to login with their existing institutional credentials' ),
+				'checked' => get_network_option( null, 'network_checklist_configure_sso', false ),
+			];
+			array_splice( $items, 3, 0, [ $sso_item ] );
+		}
+		if ( is_plugin_active( 'pressbooks-cas-sso/pressbooks-cas-sso.php' ) ) {
+			$sso_item = [
+				'title'       => __( 'Configure Single Sign On (SSO)', 'pressbooks' ),
+				'link'        => network_admin_url( 'admin.php?page=pb_cas_admin' ),
+				'description' => __( 'Allow users to login with their existing institutional credentials' ),
+				'checked'     => get_network_option( null, 'network_checklist_configure_sso', false ),
+			];
+			array_splice( $items, 3, 0, [ $sso_item ] );
+		}
+
+		// Check if Network Analytics plugin is activated
```

Comment:
@arzola I added some conditional logic so that these menu items will only be displayed to users on networks that we host -- open source users won't see the invitation to join the private forum for our clients or the link to schedule a meeting with Amy :joy: